icubam/analytics/preprocessing.py

```diff
@@ -1,15 +1,10 @@
 import itertools
-import json
 import logging
-from typing import Optional
 
 import numpy as np
 import pandas as pd
 
-from icubam.predicu.data import (
-  ALL_COLUMNS, BEDCOUNT_COLUMNS, CUM_COLUMNS, DATA_PATHS, NCUM_COLUMNS,
-  format_data
-)
+import icubam.analytics.data as data
```

Comment:
data is a really bad module name. For instance, `data.CUM_COLUMNS` in a function that processes some data makes me thing that `data` is a dataframe. I think the previous imports were clearer.

Reply:
Good point, we can refactor to another name, but IIRC data was the original module name...  I'm not super keen on direct imports into the local namespace as it gets hard to follow what comes from where when you're not super familiar with the code however.  Do you have a better name suggestion?

Reply:
Maybe `datasets.py`?

Reply:
Done